sdk/compiler/damlc/daml-doc/src/DA/Daml/Doc/Transform/Annotations.hs

```diff
@@ -84,14 +139,19 @@ applyHide = concatMap onModule
             | ADTDoc{..} <- x, all (isHide . ac_descr) ad_constrs = pure x{ad_constrs = []}
             | otherwise = [x]
 
-
-getAnn :: Maybe DocText -> [T.Text]
-getAnn = maybe [] (T.words . unDocText)
+-- Returns first line words and rest of lines
+getAnn :: Maybe DocText -> ([T.Text], [T.Text])
+getAnn mDoc = fromMaybe ([], []) $ do
+  doc <- mDoc
+  (firstLine, rest) <- uncons $ T.lines $ unDocText doc
+  pure (T.words firstLine, rest)
 
 isHide :: Maybe DocText -> Bool
-isHide x = ["HIDE"] `isPrefixOf` getAnn x
+isHide x = ["HIDE"] `isPrefixOf` fst (getAnn x)
 
-isMove :: Maybe DocText -> Maybe Modulename
+isMove :: Maybe DocText -> Maybe (Modulename, Maybe DocText)
 isMove x = case getAnn x of
-    "MOVE":y:_ -> Just (Modulename y)
+    ("MOVE":y:restFirst, restLines) ->
```

Comment:
If we make the improvements detailed in my comment for `DeclData`, we could simplify isHide to use the same implementation as isMove, and we could simplify `getAnn` to only return the first word of the first line